tests/test_schema.py

```diff
@@ -289,6 +289,23 @@ class Outer(Schema):
     assert Outer().load({"list1": val, "list2": val}) == {"list1": [], "list2": []}
 
 
+@pytest.mark.parametrize(
+    "val",
+    (
+        {"inner": {"name": "name"}, "unknown": 1},
+        {"inner": {"name": "name", "unknown_nested": 1}, "unknown": 1},
+    ),
+)
+def test_load_unknown(val):
+    class Inner(Schema):
```

Comment:
I think it would be better to explicitly set:
```python
class Meta:
    unknown = RAISE
```
for both classes here

Reply:
Thank your for the review @SVilgelm

Code updated